src/lua/functions/core/game/lua_enums.cpp

```diff
@@ -804,6 +805,28 @@ void LuaEnums::initItemTypeEnums(lua_State* L) {
 	registerEnum(L, ITEM_TYPE_UNASSIGNED);
 }
 
+void LuaEnums::initFluidEnums(lua_State* L) {
+	registerEnum(L, FLUID_NONE);
+	registerEnum(L, FLUID_WATER);
+	registerEnum(L, FLUID_WINE);
+	registerEnum(L, FLUID_BEER);
+	registerEnum(L, FLUID_MUD);
+	registerEnum(L, FLUID_BLOOD);
+	registerEnum(L, FLUID_SLIME);
+	registerEnum(L, FLUID_OIL);
+	registerEnum(L, FLUID_URINE);
+	registerEnum(L, FLUID_MILK);
+	registerEnum(L, FLUID_MANA);
+	registerEnum(L, FLUID_LIFE);
+	registerEnum(L, FLUID_LEMONADE);
+	registerEnum(L, FLUID_RUM);
+	registerEnum(L, FLUID_FRUITJUICE);
+	registerEnum(L, FLUID_COCONUTMILK);
+	registerEnum(L, FLUID_MEAD);
+	registerEnum(L, FLUID_TEA);
+	registerEnum(L, FLUID_INK);
```

Comment:
Since the enum is sequential, we can use the magic enum:
```suggestion
	for (auto value : magic_enum::enum_values<Fluids_t>()) {
		registerEnumClass(L, value);
	}
```